### PR TITLE
Upgrade deps

### PR DESCRIPTION
**Pull request by SimenB**
Only code changes are for prom-client.

I haven't tested this, but the only things I'm not really sure about are the db stuff and `install`.

I thought it might be nice to upgrade all deps before 3.0 stable release.

### CODE REVIEW

File: lib/metrics.js [hunk 1]
```diff
@@ -9,14 +9,14 @@ exports.startMonitoring = (enable, eventBus) => {
...
 
-    const requestDuration = new client.Summary(
-        'http_request_duration_milliseconds',
-        'App response time',
-        ['path', 'method', 'status'],
-        {
-            percentiles: [0.1, 0.5, 0.9, 0.99],
-        }
-    );
+    client.collectDefaultMetrics();
```
Comment (SimenB) [inline lib/metrics.js]: We want this, right?

Comment (sveisvei) [inline lib/metrics.js]: Would think so

Comment (ivarconr) [inline lib/metrics.js]: indeed!

**Comment by ivarconr** [comment]
The test verifies almost all db-stuff. The only thing left to verify is that it is possible to roll down a version.

**Pull request merged**